tests/replay/.gitignore

```diff
@@ -0,0 +1,3 @@
+package-lock.json
+replay.js
+db/
```

Comment:
Should this be combined with our main `.gitignore`?

Reply:
I don't think so but I guess depends on how you look at it.

It's just a helper script that can be used to check that the tracing module works correctly with web3. It's not strictly part of the functional tests.

Aaand it needs more work to be more configurable.